### PR TITLE
[Dark Mode]

### PR DESCRIPTION
**Pull request by BlackMix**
That pr is dark mode version
more info is here: #131

### CODE REVIEW

**Comment by lauthieb** [comment]
Thanks a lot @BlackMix for these contributions! 
I've made a tweet for that: https://twitter.com/lauthieb/status/1248575756957110273?s=20

It could be interesting to use instead the `@media (prefers-color-scheme: dark)` supported by Chromium and use the `systemPreferences.isDarkMode()` with Electron. What do you think?

I think the OS should host our preference for dark mode like other applications did.

Feel free to give me your opinion, and thanks a lot again.

File: src/renderer/globals.scss [hunk 1]
```diff
@@ -5,12 +5,20 @@
...
+$orange: #dd6300;
+$orange-invert: findColorInvert($orange);
+
 $colors: (
   "white": ($white, $black),
   "black": ($black, $white),
   "light": ($light, $light-invert),
   "dark": ($dark, $dark-invert),
   "primary": ($primary, $primary-invert),
+  "purple": ($purple, $purple-invert),
```
Comment (lauthieb) [inline src/renderer/globals.scss:20]: Why purple? Could we keep our Design System in black mode with our blue? Or our blue a little darken?

File: src/renderer/globals.scss [hunk 2]
```diff
@@ -5,12 +5,20 @@
...
+  "orange": ($orange, $orange-invert),
```
Comment (lauthieb) [inline src/renderer/globals.scss:21]: I think those numbers could be the inverted of black, so white no?

Comment (BlackMix) [reply src/renderer/globals.scss:21]: finem i'll edit

**Comment by BlackMix** [comment]
fixed colors @lauthieb

**Comment by BlackMix** [comment]
> 
> 
> Thanks a lot @BlackMix for these contributions!
> I've made a tweet for that: https://twitter.com/lauthieb/status/1248575756957110273?s=20
> 
> It could be interesting to use instead the `@media (prefers-color-scheme: dark)` supported by Chromium and use the `systemPreferences.isDarkMode()` with Electron. What do you think?
> 
> I think the OS should host our preference for dark mode like other applications did.
> 
> Feel free to give me your opinion, and thanks a lot again.

better not use it, every time electron deprecate  something and I see lots deprecated functions, but in Vuedevtools etc.. better manual

**Pull request closed without merging**